### DISCUSSION TITLE
Problem with tokio version to be used for building a sample application

### DISCUSSION THREAD

**Discussion by nsengupta** [slawlor/ractor#128] [Q&A]
In my application's `Cargo.toml`, I have used `tokio` this way:

```
ractor = "0.8.4"
async-trait = "0.1"
clap = { version = "4", features = ["derive"] }
env_logger = "0.10"
log = "0.4"
tokio = { version = "1", features = ["rt", "time", "sync", "macros", "rt-multi-thread", "signal"] }
```
This builds w/o any problem.

But, changing `tokio' dependency to just this:

```
tokio = "1.29.1"
```
 the build fails ( file attached -> app-build.output.txt [app-build-output.txt](https://github.com/slawlor/ractor/files/12024288/app-build-output.txt) )

Just to give it a shot, I have tried to build `ractor` separately, with the following `Cargo.toml` entries:

```
[ dependencies ]
tokio = { version = "1.29.1" }

[dev-dependencies ]
tokio = { version = "1.29.1" }
```
the build fails ( file attached -> ractor-build-output.txt [ractor-build-output.txt](https://github.com/slawlor/ractor/files/12024382/ractor-build-output.txt) ).

I am not an expert in Cargo entries. Also, I want to build an application on `ractor`, assuming that all dependencies  of `ractor` are clearly specified. I have gone ahead with mentioning the  `features` for `tokio`.

May be you are aware of this already, yet I thought I should bring it up and seek a solution.

**Comment by slawlor** [comment]
Both files are empty for me so I can't see the errors reported. Can you make either a paste/gist or put the error statements into code blocks in the discussion?
